recipe/tests/test_ldd.py

```diff
@@ -0,0 +1,36 @@
+import ctypes
+import sys
+from pathlib import Path
+
+import psutil
+
+# load every `libmpi*` shared library
+for libmpi in (Path(sys.prefix) / "lib").glob("libmpi*.so"):
+    print(f"Loading {libmpi}")
+    ctypes.CDLL(libmpi)
```

Comment:
Mmm... just in case, shouldn't you keep the library handle stored in a list to prevent unloading? I remember having issues elsewhere related to this, but maybe it was a Windows-only thing.

Reply:
doesn't seem to be needed in this test, but makes sense just to be sure.